src/main/java/raisetech/student/management/controller/StudentController.java

```diff
@@ -89,9 +99,9 @@ public List<StudentDetail> getActiveStudentDetailList() {
       }
   )
   @GetMapping("/students/{studentId}")
-  public StudentDetail viewStudentDetail(@PathVariable("studentId") Id studentId)
+  public StudentDetailResponse viewStudentDetail(@PathVariable("studentId") Id studentId)
       throws InvalidIdException {
-    return service.searchStudentDetail(studentId);
+    return fromDomain(service.searchStudentDetail(studentId));
```

Comment:
めちゃくちゃ個人的な好みですがstaticメソッドは呼び出し元のクラス名もあったほうが、privateメソッドの呼び出しと区別しやすいです。
```suggestion
    return StudentDetailResponse.fromDomain(service.searchStudentDetail(studentId));
```

Reply:
IntelliJ上ではstaticメソッドを斜体で表現してくれるため、読み手側からみた自クラスメソッドとの区別について意識していませんでした💦　ご指摘ありがとうございます。修正→[9f6ad43](https://github.com/saway261/StudentManagement/pull/10/commits/9f6ad43d7eb6689aa1fd26fefff24486f01ff6cb),[a76a7a6](https://github.com/saway261/StudentManagement/pull/10/commits/a76a7a63d68caf3c44d1131869d3426dc8602f6e)

assertThatのような、登場頻度が高くライブラリや標準APIが用意しているものはそのままにしたのですが、それは伝わるでしょうか？

Reply:
> assertThatのような、登場頻度が高くライブラリや標準APIが用意しているものはそのままにしたのですが、それは伝わるでしょうか？

おっしゃるとおりです！
なので今回のコメントは私の好みです 😅